tests/Jwt/FirebaseParserTest.php

```diff
@@ -51,32 +50,15 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
```

Comment:
`setExpectedException()` has, but 5.2 deprecates it. `expectException()` has not; it was added in 5.2. The latter is what Sebastian is now recommending.